addons/website_sale/static/src/js/website_sale.js

```diff
@@ -97,8 +97,13 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
         });
 
         // This allows conditional styling for the filmstrip
-        if (isBrowserFirefox() || hasTouch()) {
-            this.el.querySelector('.o_wsale_filmstip_container')?.classList.add('o_wsale_filmstip_fancy_disabled');
+        const filmstripContainer = this.el.querySelector('.o_wsale_filmstip_container');
+        const filmstripContainerWidth = filmstripContainer?.getBoundingClientRect().width;
+        const filmstripWrapperWidth = this.el.querySelector('.o_wsale_filmstip_wrapper')
+            ?.getBoundingClientRect().width;
+
+        if (isBrowserFirefox() || hasTouch() || filmstripWrapperWidth < filmstripContainerWidth) {
```

Comment:
`filmstripWrapperWidth` could be `undefined`.

Reply:
Done with the changes! :smile: